.github/release-drafter.yml

```diff
@@ -26,8 +26,10 @@ exclude-labels:
   - question
   - QA
 
-change-template: '- $TITLE (#$NUMBER)'
+change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
```

Comment:
We had the author in the past and decided to remove it as not useful information

Reply:
I guess it's a taste,  I can remove it

Reply:
I usually like to see who's behind a pull request but  I guess it's a personal taste

Reply:
I removed the author